metaconfig/src/main/scala/org/scalafmt/util/BuildTime.scala

```diff
@@ -0,0 +1,18 @@
+package org.scalafmt.util
+
+import scala.annotation.compileTimeOnly
+import scala.meta._
+
+@compileTimeOnly("Gets expanded away")
```

Comment:
Oh, so that's what that annotation is for. I'll admit I didn't really understand it.

Fixed.